src/applications/static-pages/covid-vaccine-updates-cta/createCovidVaccineUpdatesWidget.js

```diff
@@ -3,8 +3,9 @@ import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 
 export default function createCovidVaccineUpdatesWidget(store, _widgetType) {
-  const isCovidVaccineUpdatesPage =
-    document.location.pathname === '/health-care/covid-19-vaccine/';
+  const isCovidVaccineUpdatesPage = document.location.pathname.includes(
+    '/health-care/covid-19-vaccine',
+  );
```

Comment:
I think we should use strict paths for this inside of just `document.location.pathname.includes` because there could be child pages underneath we don't want to touch (could result in a runtime error with unknown effects). Maybe this instead? 

```js
const covidVaccineUpdatesPaths = new Set([
    '/health-care/covid-19-vaccine/',
   '/health-care/covid-19-vaccine-esp/',  // assuming this is the spanish path
]);

const isCovidVaccineUpdatesPage = covidVaccineUpdatesPaths.has(document.location.pathname);
```

Reply:
done